modules/standard/IO.chpl

```diff
@@ -5404,9 +5404,8 @@ proc fileWriter.writeBytes(b: bytes, size = b.size) throws {
    :arg ptr: a :class:`~CTypes.c_ptr` to some valid memory
    :arg numBytes: the number of bytes to write
 
-   :throws UnexpectedEofError: Thrown if the write operation exceeds the
-                               `fileWriter`'s specified range.
-   :throws SystemError: Thrown if an error occurred while writing to the ``fileWriter``
+   :throws UnexpectedEofError: Thrown if the write operation exceeds the ``fileWriter``'s specified range.
```

Comment:
I agree that this looks nicer, but I think we've been trying to wrap docs at 80 characters whenever possible (at least in the IO module). Would you mind switching these back?

Reply:
When I wrap a `:throws` that isn't followed by another, I get a docs warning that causes CI to fail. This happens whether I align wrapping with the start of `:throws` or with the start of the description. What's the correct way to wrap it?

Reply:
This particular instance did not fail CI, because the line I un-wrapped was not the last `:throws` of its list; it seems to only fail if you wrap the final `:throws` in a list of them.

Reply:
Commit [e623a5a](https://github.com/chapel-lang/chapel/pull/21438/commits/e623a5a5bfffe6bce7e4554933125906b2fb3790) on this PR is an example of code that has the failure I mentioned. I fixed it in the following commit [f2d62d5](https://github.com/chapel-lang/chapel/pull/21438/commits/f2d62d5f0d097ce50eebca396863f2dfe7808d66) by un-wrapping.

Reply:
Ok I see, thanks for clarifying. That's sort of an annoying quirk. I'll see if I can find a workaround, otherwise I'll probably open an issue about it.

Reply:
Sounds good. If it's acceptable to you I'll merge this PR without wrapping, and leave fixing wrapping as a future task since there are many other instances of overlong documentation lines that I didn't touch in this or my previous PR.